src/apis/terms.ts

```
@@ -0,0 +1,68 @@
+  userVerificationCode: string
+): Promise<boolean> => {
+  try {
+    if (verificationCode === userVerificationCode) {
+      return true;
+    } else {
+      return false;
+    }
+  } catch (error) {
+    throw error;
+  }
+};
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)